src/features/configure/vault/polygon_pools.js

```diff
@@ -33,6 +33,41 @@ export const polygonPools = [
     buyTokenUrl:
       'https://quickswap.exchange/#/swap?inputCurrency=0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619&outputCurrency=0xfbdd194376de19a88118e84e279b977f165d01b8',
   },
+  {
+    id: 'cometh-matic-psp',
+    name: 'MATIC-PSP LP',
+    token: 'MATIC-PSP LP',
+    tokenDescription: 'Cometh',
+    tokenAddress: '0xe3ae064907e9d49c7355504c25f8cae2e7831501',
```

Comment:
@ongmorel Can you please checksum this address?

Reply:
done